etl/pypasar/omop/death.py

```diff
@@ -22,15 +22,74 @@ def execute(self):
             raise err
 
     def initialize(self):
-        pass
+        with self.engine.connect() as connection:
+            with connection.begin():
+                # Set schema
+                connection.execute(
+                    text(f'SET search_path TO {os.getenv("POSTGRES_OMOP_SCHEMA")}'))
+
+                # Verify that each person has a unique death_date
+                result = connection.execute(
+                    text(f'''SELECT (
+                        case when count(distinct(anon_case_no, death_date))=count(distinct(anon_case_no)) then 'Unique' else 'Duplicate' end)
+                        FROM {os.getenv("POSTGRES_SOURCE_SCHEMA")}.info WHERE death_date IS NOT NULL'''
+                    )).mappings().all()[0]
+                if result['case'] != 'Unique':
+                    raise ValueError(f'There is duplicate in death_date.')
+
+                # Drop stg__death view if exists
+                connection.execute(text("DROP VIEW IF EXISTS stg__death"))
+
+                # Delete death table
+                connection.execute(text("TRUNCATE TABLE death"))
 
     def process(self):
-        # In batches
-        # Read from source
-        # Transform
-        # Ingest into OMOP Table
-        pass
+        # Set SCHEMA
+        omop_schema = os.getenv("POSTGRES_OMOP_SCHEMA")
+        source_schema = os.getenv("POSTGRES_SOURCE_SCHEMA")
+
+        with self.engine.connect() as connection:
+            with connection.begin():
+                # Read from source and create a staging table (stg__death)
+                connection.execute(
+                    text(f'''CREATE OR REPLACE VIEW {omop_schema}.stg__death AS
```

Comment:
Could create temporary view as it will be dropped automatically when the connection is closed

Reply:
stg__death is now a temporary view and will be dropped upon connection closed